lib/ocpp/v201/charge_point.cpp

```diff
@@ -719,9 +719,9 @@ void ChargePoint::handle_message(const EnhancedMessage<v201::MessageType>& messa
 
 void ChargePoint::message_callback(const std::string& message) {
     auto enhanced_message = this->message_queue->receive(message);
+    enhanced_message.message_size = message.size();
```

Comment:
Just wondering if we would like to do this assignment in the queue itself so 1.6 can use it as well.